lib/clients/reports.ex

```diff
@@ -68,4 +68,84 @@ defmodule GlassFactoryApi.Clients.Reports do
       {:error, message} -> raise message
     end
   end
+
+  @doc """
+  List rates and costs reports from the given client id
+
+  ## Examples
+
+      iex> GlassFactoryApi.Clients.Reports.list_rates_and_costs_reports(1234, [], config)
+      {:ok, [%RatesAndCostsReport{
```

Comment:
I wonder if there's a simple way of applying `mix format` to these code samples too 🤔